peerconnection.go

```diff
@@ -1079,15 +1171,119 @@ func (pc *PeerConnection) drainSRTP() {
 				return
 			}
 
-			_, ssrc, err := srtpSession.AcceptStream()
+			r, ssrc, err := srtpSession.AcceptStream()
 			if err != nil {
 				pc.log.Warnf("Failed to accept RTP %v", err)
 				return
 			}
 
-			if !handleUndeclaredSSRC(ssrc) {
-				pc.log.Warnf("Incoming unhandled RTP ssrc(%d), OnTrack will not be fired", ssrc)
-			}
+			pc.mu.Lock()
+			pc.pendingReadStreamsSRTP[ssrc] = r
+			pc.mu.Unlock()
+
+			go func() {
+				// test first N (10) packets, if no mid and rid are found give up
```

Comment:
Is this the common way other impls determine whether a stream has a known rid? Is there nothing about the track that can be used to determine the rid without reading its packets first? So I may lose the first 10 packets?

Reply:
Chrome and firefox send the mid/streamID header extensions in every rtcp packet so we will skip the first packet. Note that the first packet was also previously skipped in single stream tracks: https://github.com/pion/webrtc/blob/v2.2.12/peerconnection.go#L896